app/Http/Controllers/AuthController.php

```diff
@@ -8,6 +8,19 @@
 
 class AuthController extends Controller
 {
+    /**
+     * Assuming this method is behind the `auth.cas.force` middleware, this function will ensure that the user is
```

Comment:
How does it enforce that with just a redirect to the previous page?

Reply:
There's an expectation that you'll use this controller with appropriate middleware.  I added the `auth.cas.force` middleware to the route that uses this controller.

That being said, I've noticed it can cause a redirect loop if the user is unauthenticated so I need to add some extra logic to catch those situations.

Reply:
The redirect loop bug should be fixed now; it'll only redirect for the OAuth routes where this was needed